test/attr/ApplicationMain/attr_main_struct_available_future.swift

```diff
@@ -3,8 +3,7 @@
 // REQUIRES: OS=macosx
 
 @main // expected-error {{'main()' is only available in macOS 10.99 or newer}}
-@available(OSX 10.0, *)
-struct EntryPoint {
+@available(OSX 10.0, *) struct EntryPoint { // expected-note {{update existing @available attribute}} {{16-20=10.99}} 
```

Comment:
I'm not sure why but adding expected fixit note like:
```
@available(OSX 10.0, *) // expected-note {{16-20=10.99}}
```
didn't work. (maybe because the original note is from the `struct EntryPoint`line?)
So I put the two expressions in the same line, but is there a proper way to fix this?